statsmodels/tsa/stattools.py

```diff
@@ -635,40 +646,79 @@ def pacf_burg(x, nlags=None, demean=True):
     return pacf, sigma2
 
 
-#NOTE: this is incorrect.
```

Comment:
This comment has been a huge bother.  Very glad to see it go.

Reply:
It is tested to clearly correct